boxoffice/static/js/templates/org.html.js

```diff
@@ -1,7 +1,7 @@
 export const orgTemplate = `
   <div class="container">
     <div class="row">
-      <h1 class="header">{{ name }} Item collections</h1>
+      <h1 class="header">{{ title }} Item collections</h1>
```

Comment:
1. Should be "item collections" (lowercase i) since it's a sentence.
2. I'm not comfortable with title being included in a sentence like this since (a) it makes localization hard and (b) it assumes the final sentence will be grammatically correct.